### ISSUE TITLE
The grid system

### ISSUE THREAD

**Issue by joseluis** [andamira/medula#40]
These are grids I happen to like the most:

- [Susy](http://susy.oddbird.net/)
- [Jeet](http://jeet.gs/)
- [Neat](http://neat.bourbon.io/)
- [Gridle](http://gridle.org/)

I'd very much prefer prefer not to use markup classes. It's cleaner and more versatile. Since everyone is probably used to a different system, I wouldn't mind to give a choice as long as the theme remains clean and simple.

Right now Susy is the default grid, but the integration can definitely be improved. And I'm seriously considering changing the default grid to Jeet.

**Comment by jan-dh** [comment]
I think a choice for one of these (I think Susy has the biggest fanbase atm) would be valid, providing you document on how to swap one out for the other.

**Comment by YemSalat** [comment]
Yep, I also vote for `Susy` and ability to switch it to something else. From your list I only used Susy, but in recent projects I switched to my own set of mixins and I would like to be able to continue using them with the new starter theme.

**Comment by joseluis** [comment]
I basically agree.

* What I like about Susy is its power and the big user base.
* What I like about Jeet it seems powerful enough while being easier to read and comprehend.

Some resources:
* http://corysimmons.github.io/presentations/jeet-5/

**Comment by joseluis** [comment]
I think I'll wait at least for the [release of Jeet 7](https://github.com/mojotech/jeet/issues/425) because in the current version 6 it [outputs and excessive amount of duplicated CSS code](https://github.com/mojotech/jeet/issues/372). And its future is a little uncertain.

**Comment by joseluis** [comment]
Just discovered [Lost grid](https://github.com/corysimmons/lost) and I'm liking it very much.

It's different from the others in that it's a postcss plugin, and it's based on calc(). The syntax is very clean, very simple, and I believe it fits well with Medula spirit.
